Item/Item.module.less

```diff
@@ -72,13 +76,11 @@
 			.labelAfter,
 			.labelBefore {
 				.content {
-					box-sizing: border-box;
 					min-width: @agate-item-content-width;
 				}
 
 				.label {
-					box-sizing: border-box;
-					max-width: @agate-item-label-max-width;
+					max-width: @agate-item-label-width;
```

Comment:
Why did you change this?

Reply:
My mistake, I will change it back to `@agate-item-label-max-width`